lib/helper/resultHelper.py

```diff
@@ -161,35 +161,26 @@ def fps_cal(file_path, default_fps):
     fps_stat = "0"
     fps_return = 0
     tolerance = 0.03  # recording fps tolerance is set to 3%
+    fps_frequency_threshold = 0.9  # fps frequency should be more than 90%
     if os.path.exists(file_path):
         with open(file_path, 'r') as fh:
             data = ''.join([line.replace('\n', '') for line in fh.readlines()])
             fps = map(int, re.findall('fps=(\s\d+\s)', data))
             count = Counter(fps)
         fh.close()
-        all_fps = count.most_common()
-        total_record = sum([item[1] for item in all_fps])
-        default_fps_element = [item for item in all_fps if item[0] == default_fps]
-        mode_fps = count.most_common()[0]
-        if not default_fps_element and \
-                (float(mode_fps[1]) / total_record) >= 0.9 and \
-                round(default_fps * (1 - tolerance)) <= mode_fps[1] <= round(default_fps * (1 + tolerance)):
-            fps_return = mode_fps[0]
-        elif mode_fps[0] == default_fps:
-            if (float(mode_fps[1]) / total_record) >= 0.9:
-                fps_return = mode_fps[0]
-            else:
-                if len(all_fps) > 1 and \
-                        float(mode_fps[1] + all_fps[1][1]) / total_record >= 0.9 and \
-                        round(default_fps * (1 - tolerance)) <= all_fps[1][0] <= round(default_fps * (1 + tolerance)):
-                    fps_return = round(mode_fps[0] - float(mode_fps[0] - all_fps[1][0]) *
-                                       mode_fps[1] / total_record)
-        elif round(default_fps * (1 - tolerance)) <= mode_fps[0] <= round(default_fps * (1 + tolerance)) and \
-                float(default_fps_element[0][1] + mode_fps[1]) / total_record >= 0.9:
-            fps_return = round(default_fps_element[0][0] - float(default_fps_element[0][0] - mode_fps[0]) * default_fps_element[0][1] / total_record)
-        else:
-            fps_return = mode_fps[0]
+        all_fps_tuple = count.most_common()
+        if not all_fps_tuple:
+            logger.error("Cannot get fps information from log.")
             fps_stat = "1"
+        else:
+            total_record = sum([item[1] for item in all_fps_tuple])
```

Comment:
adding comment here to record `item[0] is fps` and `item[1] is counter` will be helpful.